README.md

```diff
@@ -1,6 +1,31 @@
 DVDOrderAgent.bundle
 ====
 
-code was taken from the plex [forums](https://forums.plex.tv/discussion/21141/dvd-order-agent)  
-this was the newest version I could find in the mess of that thread  
-I did very slightly modify this for use with my [PlexThemeMusic.bundle](https://github.com/HuggableSquare/PlexThemeMusic.bundle) fork  
+The code for this agent was taken from the plex [forums](https://forums.plex.tv/discussion/21141/dvd-order-agent),
+this was the newest version I could find in the mess of that thread.
+
+I did very slightly modify this for use with my [PlexThemeMusic.bundle](https://github.com/HuggableSquare/PlexThemeMusic.bundle) fork.
+
+Installation
+----
+
+Add this bundle to your PMS plugins. On a Mac that will look like this:
+
+```bash
+cd "~/Library/Application Support/Plex Media Server/Plug-ins/"
+git clone https://github.com/HuggableSquare/DVDOrderAgent.bundle
+```
```

Comment:
Hi! I definitely appreciate the PR, I'm thinking that a good idea would be to add these Mac specific instructions to the [wiki](https://github.com/HuggableSquare/DVDOrderAgent.bundle/wiki/Installation-Instructions) and then add a link to the wiki here instead. Something like: "Add this bundle to your PMS plugins. Instructions for that can be found on the [wiki](https://github.com/HuggableSquare/DVDOrderAgent.bundle/wiki/Installation-Instructions)." That way, people can update the instructions without having to submit PRs. Past that, everything else looks good to me, if you're willing to commit that change, then I'd be down to merge the PR. Thanks!